ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -91,12 +106,42 @@ export default class ConfirmPageContainerContent extends Component {
       ethGasPriceWarning,
     } = this.props;
 
+    const isDisabled = () => {
+      return this.state.confirmAnyways ? false : disabled;
+    };
+
     return (
       <div className="confirm-page-container-content">
         {warning ? <ConfirmPageContainerWarning warning={warning} /> : null}
         {ethGasPriceWarning && (
           <ConfirmPageContainerWarning warning={ethGasPriceWarning} />
         )}
+        {hasSimulationError && (
+          <div className="confirm-page-container-content__error-container">
+            <ActionableMessage
+              type="danger"
+              message={this.context.t('simulationErrorMessage', [
+                <Button
```

Comment:
The button does not need to be a child of the translation string. See `metamask-extension/ui/components/ui/actionable-message/actionable-message.stories.js` for how you can use the `primaryAction` property of the `ActionableMessage` component to specify the label and action for the button

Reply:
HI Dan, I tried that, but I had 2 issues with it

1.  I couldn't get the primary action to fire the event correctly. Basically the state is not changing on click.
2. I couldn't get the button to be in red :(

I saw another component use this method though.

Reply:
okay, got it working